cdap-docs/_common/common-build.sh

```diff
@@ -60,6 +60,7 @@ EOF`
 
 SCRIPT=`basename ${0}`
 SCRIPT_PATH=`pwd`
```

Comment:
Just noticed this expects the script is always run from the correct working directory, no?